src/test/java/ro/developmentfactory/Cinema/SeatsAlreadyReservedExceptionTest.java

```diff
@@ -0,0 +1,55 @@
+package ro.developmentfactory.Cinema;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class SeatsAlreadyReservedExceptionTest {
```

Comment:
you don't need to write separate tests for your custom exception only, it's covered in `CinemaServiceTest` class